antarest/study/dao/hybrid_dao.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+from typing import Sequence
+
+from sqlalchemy.orm import Session
+from typing_extensions import override
+
+from antarest.study.business.model.link_model import Link
+from antarest.study.dao.db_link_dao import DbLinkDao
+from antarest.study.dao.file.file_study_dao import FileStudyTreeDao
+from antarest.study.storage.rawstudy.model.filesystem.factory import FileStudy
+
+
+class HybridDao(FileStudyTreeDao):
```

Comment:
We need to decide the implementation strategy, in particular if we want this kind of hybrid storage or not:
to be honest I think it will make things more complex. But otherwise, we are more or less forced to implement things in a given order, starting with areas.

As discussed:
- one of the first thing we need to do, is also to implement an import feature where we can choose the target DAO (like, simply adding an option in the import API). This way we will be able to use the feature in the app
- we should probably also have a "feature flag" in the configuration to enable or not that feature, until it's completely implemented